tests/RobotFramework/tests/plugin_apt/nfpm-version1.yaml

```diff
@@ -0,0 +1,15 @@
+name: sampledeb
+arch: all
+platform: linux
+version: 1.0.0
+section: default
+priority: optional
+maintainer: Your Name <your.email@example.com>
+description: My Sample App Debian Package
+homepage: https://example.com/myapp
+license: MIT
+
+contents:
+  - src: ./sampledeb_1/etc/sampledeb/sampledeb.cfg
```

Comment:
The nfpm file is referencing files that are not present in the project.

You don't need to reflect the target destination in the `src` field. You can just add configuration files for each version, e.g. `sampledeb_1.cfg` and `sampledeb_2.cfg`.

Reply:
added `sampledeb_1.cfg` file

Reply:
Yes, though unfortunately the files you added don't match your nfpm config which resulted in the errors when trying to build the debian packages via the build.sh script.

I've fixed the config (and made some formatting improvements in the robot file)